redom/android/src/main/java/com/pushtorefresh/redom/android/recycler/Adapter.kt

```diff
@@ -1,18 +1,9 @@
 package com.pushtorefresh.redom.android.recycler
 
 import android.view.ViewGroup
-import androidx.appcompat.widget.AppCompatButton
-import androidx.appcompat.widget.AppCompatCheckBox
-import androidx.appcompat.widget.AppCompatTextView
-import androidx.appcompat.widget.SwitchCompat
+import androidx.appcompat.widget.*
```

Comment:
Do we want to use stars? I think an explicit declaration is better.

Reply:
I agree, we can address this in separate issue now that we have codestyle #43